protocol/0064-VALP-validator_performance_based_rewards.md

```diff
@@ -69,10 +69,9 @@ The performance score should be available on all the same API endpoints as the `
   * Verify that at the beginning of the next epoch the validator has non 0 performance score, and voting power is greater than 10. 
   * Update the network parameter `reward.staking.delegation.minimumValidatorStake` for minimum self-stake to be more than is self-delegated. 
   * Verify that, at the beginning of the next epoch, all performance scores are 0 and voting power for all is 1 but the network keeps producing blocks and no nodes were removed from Tendermint.
+ 6. Scores are produced after checkpoint reload (<a name="0064-VALP-006" href="#0064-VALP-006">0064-VALP-006</a>):  
+  * Restart network from checkpoint which contains validator performance scores. After restart when network is operable, scores should be available and updated correctly to reflect current validator count  
```

Comment:
When you say checkpoint, do you mean LNL checkpoint (0073-LIMN-limited_network_life.md) or snapshot? Because LNL checkpoint spec says nothing about maintaining validator performance scores.

Reply:
Updated to reference snapshot and not checkpoints